dockerfiles/README.md

```diff
@@ -82,16 +82,11 @@ python3 docker_openvino.py build --file "dockerfiles/ubuntu18/openvino_cgvh_dev_
 ```
 For data_dev/runtime/data_runtime/proprietary distributions, please set appropriate `-dist` and `--file` options.
 
-Or via Docker Engine directly, but you need specify `package_url` argument (see [Where to get OpenVINO package section](#where-to-get-openvino-package)) and OpenCL* version to support GPU:
-`GMMLIB`, `IGC_CORE`, `IGC_OPENCL`, `INTEL_OPENCL`, `INTEL_OCLOC`
+Or via Docker Engine directly, but you need specify `package_url` argument (see [Where to get OpenVINO package section](#where-to-get-openvino-package)) and OpenCL* version with `INTEL_OPENCL` argument to support GPU:
```

Comment:
add something like this "to natively support inference on 11th Generation Intel® Core™ Processor Family for Internet of Things (IoT) Applications (formerly codenamed Tiger Lake) from OpenVINO Docker container"